jkube-kit/jkube-kit-quarkus/src/test/java/org/eclipse/jkube/quarkus/generator/QuarkusGeneratorTest.java

```diff
@@ -465,21 +465,21 @@ void withManualConfigAndFastJarAndLegacyInTarget_shouldReturnAssemblyForQuarkusA
         .hasFieldOrPropertyWithValue("targetDir", "/deployments")
         .hasFieldOrPropertyWithValue("excludeFinalOutputArtifact", true)
         .extracting(AssemblyConfiguration::getLayers)
-        .asList().hasSize(2)
+        .asInstanceOf(InstanceOfAssertFactories.list(Assembly.class)).hasSize(2)
         .satisfies(layers -> assertThat(layers).first().asInstanceOf(InstanceOfAssertFactories.type(Assembly.class))
           .hasFieldOrPropertyWithValue("id", "lib")
           .extracting(Assembly::getFileSets)
-          .asList().singleElement()
+          .asInstanceOf(InstanceOfAssertFactories.list(Assembly.class)).singleElement()
           .hasFieldOrPropertyWithValue("outputDirectory", new File("."))
-          .extracting("includes").asList()
+          .extracting("includes").asInstanceOf(InstanceOfAssertFactories.list(String.class))
           .containsExactly("lib"))
         .satisfies(layers -> assertThat(layers).element(1).asInstanceOf(InstanceOfAssertFactories.type(Assembly.class))
           .hasFieldOrPropertyWithValue("id", "fast-jar")
           .extracting(Assembly::getFileSets)
-          .asList().singleElement()
+          .asInstanceOf(InstanceOfAssertFactories.list(Assembly.class)).singleElement()
           .hasFieldOrPropertyWithValue("outputDirectory", new File("."))
           .hasFieldOrPropertyWithValue("excludes", Arrays.asList("lib/**/*", "lib/*"))
-          .extracting("includes").asList()
+          .extracting("includes").asInstanceOf(InstanceOfAssertFactories.list(String.class))
```

Comment:
I think maybe you can directly use `InstanceOfAssertFactories.STRING` here. Could you please check if it's working with `InstanceOfAssertFactories.STRING`?